content/en/user-guide/aws/qldb/index.md

```diff
@@ -8,13 +8,309 @@ aliases:
+
+**2. Immutable and Append-Only**: Once data is written to the QLDB ledger, it is immutable. This means that transactions cannot be modified or deleted. New transactions are appended to the ledger.
+
+**3. Cryptographically Verifiable**: QLDB uses cryptographic hashing and digital signatures to ensure the integrity of data. Clients can verify the authenticity and integrity of the data in the ledger.
+
+**4. ACID Transactions**: QLDB supports ACID (Atomicity, Consistency, Isolation, Durability) transactions, ensuring that transactions are processed reliably and securely.
+
+**5. SQL-like Query Language**: QLDB supports PartiQL, which is a SQL-compatible query language. This allows developers familiar with SQL to query the data stored in the ledger.
+
+**6. Managed Service**: AWS QLDB is a fully managed service. AWS handles the operational aspects such as server provisioning, scaling, and maintenance, allowing developers to focus on building applications.
+
+**7. Integration with AWS Services**: QLDB can be integrated with other AWS services such as AWS Lambda, Amazon S3, and AWS Step Functions, enabling developers to create powerful and scalable applications.
+
+To learn more about QLDB please refer to the [official documentation](https://docs.aws.amazon.com/qldb/).
+To find out what operations are supported by the QLDB service on LocalStack, please check the [QLDB service coverage page]({{< ref "/references/coverage/coverage_qldb/index.md" >}} "QLDB service coverage page"). 
+
+## Getting started with QLDB on LocalStack
+
+These instructions will follow along with the [getting started guide](https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started.html) from the official documentation, but instead of using the console to
+perform all the operations, the LocalStack AWS CLI (management API only) and the QLDB shell (data API only) will be used.
+
+### Installing the QLDB shell
+
+QLDB supports PartiQL, a SQL-compatible query language, which allows you to query and manipulate data stored in QLDB. 
+You can write PartiQL statements to perform complex queries, aggregations, and transformations on your data.
+Amazon QLDB provides a command line shell for interaction with the transactional data API. With the QLDB shell,
+you can run PartiQL statements on ledger data.
+
+For instructions on how to use and install the latest version of the QLDB shell, see the [README.md](https://github.com/awslabs/amazon-qldb-shell/blob/main/README.md#installation) file on GitHub. 
+QLDB provides pre-built binary files for Linux, macOS, and Windows in the [Releases](https://github.com/awslabs/amazon-qldb-shell/releases) section of the repository.
+
+For macOS, the shell integrates with the `aws/tap` Homebrew tap. To install the shell on macOS using Homebrew, run the following commands.
+
+{{< command >}}
+$ xcode-select --install # Required to use Homebrew
+$ brew tap aws/tap # Add AWS as a Homebrew tap
+$ brew install qldbshell
+{{< / command >}}
```

Comment:
If we have stated beforehand that QLDB shell is used, we don't need to provide alternative instructions on installing it. It would be better to link the `README` to the first reference of QLDB shell and ask users to have it pre-installed.

Reply:
I think this makes sense. We can remove lines 52-69